documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -148,7 +157,7 @@ Full serialization of single submodel element _Property_:
 }
 ----
 
-With the SerializationModifier set to _Value,_ the payload is minimized to the following:
+With the SerializationModifier set to _Value,_ the payload within a submodel or structured submodel element is minimized to the following:
```

Comment:
What is a **structured** submodel element? Is it something with children, i.e. a non-leaf element?